test/test.py

```diff
@@ -0,0 +1,55 @@
+from cube_fit.plot import *
```

Comment:
Why don't you add those tests to the existing test file or at least to the same `tests` folder?

Reply:
Ok, I will upload some test scripts and you can tell me how I can structure it. I think they're really just messy workings, but perhaps they will be useful for the future